telemetry/generated/page.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
Despite what the above comment says, we will be committing these files for now. If we ignore them, the CI build step fails.

Reply:
That is bizarre. Do we know the exact reason why?

Reply:
@jpezninjo yes because we need to setup the CI to generate them. However when @wtfluckey and I talked we discussed committing these for now because it makes first time dev easier.